interfaces/apparmor/template_vars.go

```diff
@@ -31,7 +31,8 @@ import (
 // apparmor template and by apparmor snippets.
 func templateVariables(info *snap.Info, securityTag string) string {
 	var buf bytes.Buffer
-	fmt.Fprintf(&buf, "@{SNAP_NAME}=\"%s\"\n", info.Name())
+	// TODO parallel-install: use of proper instance/store name
+	fmt.Fprintf(&buf, "@{SNAP_NAME}=\"%s\"\n", info.InstanceName())
```

Comment:
This is fine

Reply:
> This is fine. 

Somehow these strings must match each other so that the template name and the method name are the same.